tests/unit/test_tenant.py

```diff
@@ -10,6 +12,28 @@ def setup(self):
         self.custom_404_msg = "Tenant not found"
         self.schema = TenantSchema
 
+    def test_only_active_lease_returned(self, faker, create_lease, create_tenant):
+        tenant = create_tenant()
+
+        # Create an expired lease for the tenant
+        lease_expired_end = faker.date_time_this_decade(
+            before_now=True, after_now=False
+        )
+        lease_expired_start = lease_expired_end - datetime.timedelta(days=365)
+        lease_expired = create_lease(
+            tenant=tenant,
+            dateTimeStart=lease_expired_start,
+            dateTimeEnd=lease_expired_end,
+        )
```

Comment:
Maybe some day we can extract this out to a `create_expired_lease` factory, but if this is the only place we need to do this then perhaps it's not necessary.